viper/parser/parser.py

```diff
@@ -140,7 +140,7 @@ def _mk_getter_helper(typ, depth=0):
 # Make a list of getters for a given variable name with a given type
 def mk_getter(varname, typ):
     funs = _mk_getter_helper(typ)
-    return ['@constant\ndef get_%s%s(%s) -> %s: return self.%s%s' % (varname, funname, head.rstrip(', '), base, varname, tail)
+    return ['@public\n@constant\ndef get_%s%s(%s) -> %s: return self.%s%s' % (varname, funname, head.rstrip(', '), base, varname, tail)
```

Comment:
Please for the love of all things Python use the triple quotes here:
```python
return """
@public
@constant
def ....
""".format(...)

Reply:
Will do